electricitymap/contrib/lib/models/event_lists.py

```diff
@@ -227,6 +230,8 @@ def merge_production_breakdowns(
 
         df = df.drop(columns=["source", "sourceType", "zoneKey"])
         df = df.groupby(level=0, dropna=False)["data"].apply(list)
+        if matching_timestamps_only:
+            df = df[df.apply(lambda x: len(x) == len_ungrouped_production_breakdowns)]
```

Comment:
Should we also try and log the datapoints that where discarded or just the amount? Could help with debugging later on?

Reply:
Yes I think that could be a useful information, I will add it